editor/plugins/script_text_editor.cpp

```diff
@@ -76,34 +76,97 @@ void ScriptTextEditor::_load_theme_settings() {
 	text_edit->clear_colors();
 
 	/* keyword color */
```

Comment:
Could remove this comment as it no longer makes scene, same in the shade editor.